pkg/private/tar/tar_writer.py

```diff
@@ -86,10 +87,11 @@ def __init__(self,
     else:
       mode = 'w:'
       if compression in ['tgz', 'gz']:
+        compresslevel = int(compresslevel) if compresslevel or compresslevel == 0 else 6
```

Comment:
Why do you need the `compresslevel == 0`?

Reply:
Well it's indeed unnecessary... I'll remove it.